src/fix/main.rs

```diff
@@ -0,0 +1,1059 @@
+//TODO restore cargo.toml bin before shipping
+
+use std::str::FromStr;
+use clap::Parser;
+use std::io::{self, Read, Write, Seek};
+use std::os::unix::io::AsRawFd;
+use std::fs::{File, OpenOptions};
+use clap_num::maybe_hex;
+use std::process;
+
+const BANK_SIZE: usize = 0x4000;
+const OPTSTRING: &str = "Ccf:i:jk:l:m:n:Op:r:st:Vv";
+const FIX_HEADER_SUM: u8 = 0x20;
+const TRASH_HEADER_SUM: u8 = 0x10;
+const FIX_GLOBAL_SUM: u8 = 0x08;
+const TRASH_GLOBAL_SUM: u8 = 0x04;
+
+#[derive(Debug, Clone)]
+pub struct CLIOptions {
```

Comment:
I'm not going to fix this in the rest of the file so make sure you find/replace this:
```suggestion
pub struct CliOptions {
```